app/graph/mutations/update_submission_mutation.rb

```diff
@@ -49,9 +52,12 @@ def self.resolve(_obj, args, context)
         raise(GraphQL::ExecutionError, 'Submission Not Found')
       end
 
+      # FIXME: Why does the API reject this property?
+      params.delete('dimensions_metric')
```

Comment:
Not sure whats up here, but Rails will reject a submission if this property is included. Looking in Admin UI, this field isn't found, only a naked "dimension: 222x222x222". Perhaps related?